src/Microsoft.Identity.Web/AuthorizeForScopesAttribute.cs

```diff
@@ -32,38 +32,37 @@ public class AuthorizeForScopesAttribute : ExceptionFilterAttribute
         /// <summary>
         /// Scopes to request.
         /// </summary>
-        public string[] Scopes { get; set; }
+        public string[]? Scopes { get; set; }
 
         /// <summary>
         /// Key section on the configuration file that holds the scope value.
         /// </summary>
-        public string ScopeKeySection { get; set; }
+        public string? ScopeKeySection { get; set; }
 
         /// <summary>
-        /// Handles the MsalUiRequiredException.
+        /// Handles the <see cref="MsalUiRequiredException"/>.
         /// </summary>
         /// <param name="context">Context provided by ASP.NET Core.</param>
         public override void OnException(ExceptionContext context)
         {
-            // Do not re-use the attribute param Scopes. For more info: https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/issues/273
```

Comment:
why remove this comment?

Reply:
it's further down.

Reply:
It was moved few lines down.
https://github.com/AzureAD/microsoft-identity-web/blob/6fe02dbb312bed2d7351ff9035a6e717cafc7574/src/Microsoft.Identity.Web/AuthorizeForScopesAttribute.cs#L63-L64